packages/subnav/partials/CtaLinks/github-stars-link/style.module.css

```diff
@@ -1,10 +1,10 @@
 .root {
-  --stars-border-color: rgba(29, 30, 35, 0.2);
-  --scrim-opacity: 0;
+  --local-scrim-color: transparent;
+  --local-divider-color: var(--border-color); /* from theme.module.css */
```

Comment:
These `--local` vars make it possible to group styles fully by element. Without them I think we'd have to group by state, which might be okay in this specific situation, but quickly gets messy so I think should be avoided (per CSS guidelines in Notion - though as always, open to discussion on those as well).